python-package/lightgbm/dask.py

```diff
@@ -556,13 +625,13 @@ def fit(self, X, y, sample_weight=None, init_score=None, group=None, client=None
                    + ' ' * 12 + 'Dask client.\n'
                    + ' ' * 8 + _eval_set + _after_eval_set)
 
-    def predict(self, X, **kwargs):
+    def predict(self, X: _DaskCollection, **kwargs: Any) -> _DaskCollection:
```

Comment:
Based on https://github.com/microsoft/LightGBM/pull/3866/files#r565691112.
```suggestion
    def predict(self, X: _DaskCollection, **kwargs: Any) -> dask_Array:
```